terraform/modules/client-elbs/star_18f_gov_elb.tf

```diff
@@ -1,4 +1,6 @@
 resource "aws_elb" "star_18f_gov_elb" {
+  count = "${var.count}"
```

Comment:
That's what I wanted to do originally, but we're still on terraform 0.7.13, and conditionals were added in 0.8.0. We're blocked on upgrading to latest by https://github.com/hashicorp/terraform/pull/12414, but we might be able to upgrade to a verison in the 0.8.x series now. Will check.